diag_test.py

```
@@ -791,12 +837,61 @@ def auc_ci_delong(y_true: Any, y_scores: Any) -> Tuple[float, float, float]:
         return np.nan, np.nan, np.nan
 
 
+def _format_missing_data_html(missing_info):
```

Ruff (0.14.13)

Missing return type annotation for private function `_format_missing_data_html`

(ANN202)